src/main/java/io/trino/plugin/tiledb/TileDBRecordCursor.java

```diff
@@ -542,12 +541,13 @@ else if (attBounds.getSecond() == null) {
 
     /**
      * Returns the Query condition for the given bound.
+     *
      * @param attr The attribute
      * @param isString True if the attribute is String
      * @param bound The bound
      * @param op TIleDB operator
      * @return The query Condition
-     * @throws TileDBError
+     * @throws TileDBError TileDBError
```

Comment:
Why the repetition?

Reply:
Trino 411 introduced more format checks, one of them is that every parameter in the java docs needs to have a description. That's why I added TileDBError